src/main/java/jp/co/soramitsu/sora/didresolver/Controllers/DIDResolverBaseController.java

```diff
@@ -17,15 +18,14 @@
 
     public static final String PATH = "/did";
 
-    /*protected final ValidateService validateService;
+    protected StorageService storageService;
 
-    @Autowired
-    public DIDResolverBaseController(ValidateService validateService) {
-        this.validateService = validateService;
-    }*/
+    DIDResolverBaseController(StorageService storageService){
+        this.storageService = storageService;
+    }
 
     @PostMapping(consumes = {MediaType.APPLICATION_JSON_VALUE})
     public void createDDO(@Validated @RequestBody DDO ddo){
-
+        storageService.createOrUpdate(ddo.getId(),ddo);
```

Comment:
if document exists, API must return code 400, which indicates that DDO is already registered.